packages/state-transition/src/epoch/processProposerLookahead.ts

```diff
@@ -10,12 +11,26 @@ import {computeProposerIndices} from "../util/seed.js";
  * this means that at the start of epoch `N`, the proposer lookahead for epoch
  * `N+1` will be computed and included in the beacon state's lookahead.
  */
-export function processProposerLookahead(fork: ForkSeq, state: CachedBeaconStateFulu): void {
+export function processProposerLookahead(
+  fork: ForkSeq,
+  state: CachedBeaconStateFulu,
+  cache: EpochTransitionCache
+): void {
   // Shift out proposers in the first epoch
   const remainingProposerLookahead = state.proposerLookahead.getAll().slice(SLOTS_PER_EPOCH);
 
   // Fill in the last epoch with new proposer indices
-  const lastEpochProposerLookahead = computeProposerIndices(fork, state, state.epochCtx.epoch + MIN_SEED_LOOKAHEAD + 1);
+  const epoch = state.epochCtx.epoch + MIN_SEED_LOOKAHEAD + 1;
+
+  const shuffling =
+    state.epochCtx.shufflingCache?.getSync(epoch, cache.nextShufflingDecisionRoot, {
```

Comment:
don't need the last param because we handled `computeEpochShuffling()` below

Reply:
so should we track metric separately? if we pass `buildProps` we get the metrics from shuffling cache

https://github.com/ChainSafe/lodestar/blob/6fa9960988a3a56675bc0edac2594c61b359a3fe/packages/beacon-node/src/chain/shufflingCache.ts#L156-L158